### DISCUSSION TITLE
Migrate configs between versions

### DISCUSSION THREAD

**Discussion by leoniDEV** [TechnitiumSoftware/DnsServer#528] [Q&A] [answered]
I build custom docker images to deploy DNS Server in my environment.
It is safe to store configs on a volume and mount that volume to an updated version of a docker container with a newer version of DNS Server?

**Comment by ShreyasZare** [comment, marked as answer]
Thanks for asking. The config files are backwards compatible so you wont have any issues when upgrading to newer version. It is recommended to take a backup though using the option at the bottom in the Settings section.
